### PR TITLE
docs: Update docs for keyboard shortcuts

### PR DESCRIPTION
**Pull request by tonyferrell**
#### Description of Change
Updates the fiddles for Keyboard shortcuts. (local, global, and main-intercept).

The fiddles work as is so this PR:
* removes `nodeIntegration: true` from the `BrowserWindow`
* removes the version display from the home page.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation is changed or added
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: none

cc @erickzhao

### CODE REVIEW

File: docs/tutorial/keyboard-shortcuts.md [hunk 1]
```diff
@@ -81,15 +81,21 @@ If you want to handle keyboard shortcuts within a [BrowserWindow], you can
...
+  // You can put code here to handle the keypress.
+  document.getElementById("last-keypress").innerText = event.key;
+  console.log(`You pressed ${event.key}`);
+}
+
+window.addEventListener('keyup', handleKeyPress, true);
 ```
 
-Note the third parameter `true` indicates that the listener will always receive
+> Note the third parameter `true` indicates that the listener will always receive
```
Comment (erickzhao) [inline docs/tutorial/keyboard-shortcuts.md:94]:
```suggestion
> Note:  the third parameter `true` indicates that the listener will always receive
```

File: docs/tutorial/keyboard-shortcuts.md [hunk 2]
```diff
@@ -81,15 +81,21 @@ If you want to handle keyboard shortcuts within a [BrowserWindow], you can
...
 key presses before other listeners so they can't have `stopPropagation()`
 called on them.
 
-#### Intercepting events in the main process
+### Intercepting events in the main process
```
Comment (erickzhao) [inline docs/tutorial/keyboard-shortcuts.md:98]: Should probably update the Fiddle content below

Comment (erickzhao) [reply docs/tutorial/keyboard-shortcuts.md:98]: Also, I think this should still be an `####`

File: docs/fiddles/features/keyboard-shortcuts/web-apis/renderer.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+function handleKeyPress(event) {
+  // You can put code here to handle the keypress.
+  document.getElementById("last-keypress").innerText = event.key;
+  console.log(`You pressed ${event.key}`);
+}
+
+window.addEventListener('keyup', handleKeyPress, true);
```
Comment (erickzhao) [inline docs/fiddles/features/keyboard-shortcuts/web-apis/renderer.js:7]:
```suggestion
function handleKeyPress (event) {
  // You can put code here to handle the keypress.
  document.getElementById("last-keypress").innerText = event.key
  console.log(`You pressed ${event.key}`)
}

window.addEventListener('keyup', handleKeyPress, true)
```

File: docs/fiddles/features/keyboard-shortcuts/web-apis/main.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  const mainWindow = new BrowserWindow({
+    width: 800,
+    height: 600,
+  })
+
+  // and load the index.html of the app.
+  mainWindow.loadFile('index.html')
+
+  // Open the DevTools.
+  // mainWindow.webContents.openDevTools()
```
Comment (erickzhao) [inline docs/fiddles/features/keyboard-shortcuts/web-apis/main.js:16]: Maybe we should remove this commented code to avoid confusion

**Review by erickzhao — CHANGES_REQUESTED** [review]

File: docs/tutorial/keyboard-shortcuts.md (hunk 2 above)
Comment (tonyferrell) [reply docs/tutorial/keyboard-shortcuts.md:98]: Whoops, yep. Forgot to put this back.

File: docs/fiddles/features/keyboard-shortcuts/global/index.html [hunk 5]
```diff
@@ -7,10 +7,5 @@
 </head>
 <body>
     <h1>Hello World!</h1>
-    <p>
```
Comment (erickzhao) [inline docs/fiddles/features/keyboard-shortcuts/global/index.html:10]: Nit: We could describe which global shortcut we want to use here!

**Review by erickzhao — APPROVED** [review]

File: docs/fiddles/features/keyboard-shortcuts/global/index.html (hunk 5 above)
Comment (tonyferrell) [reply docs/fiddles/features/keyboard-shortcuts/global/index.html:10]: Added a description to each fiddle!

**Review by erickzhao — APPROVED** [review]

**Review by zcbenz — APPROVED** [review]

**Pull request merged**